File: prime_numbers/src/primes.js
const start = new Date().getTime();

let primes = 0;

let x = 1;

while (true) {
  const now = new Date().getTime();
  if (now - start >= 10000) {
    console.log(
      `Found ${primes} primes numbers on ${x} in ${(now - start) / 1000}s`
    );
    break;
  }

  let isPrime = true;
  let y = 3;
  while (isPrime && y <= Math.sqrt(x)) {
    if (x % y === 0) {
      isPrime = false;
    }
    y += 2;
  }
  if (isPrime) {
    primes++;
  }
  x += 2;
}

File: prime_numbers/src/primes.py
import math
import time

start = time.time()

primes = 0

x = 1

while (True):
    now = time.time()
    if now - start >= 10:
        print("Found", primes, "primes numbers on", x, "in", now - start, "s")
        break


    is_prime = True
    y = 3
    while (is_prime and y <= math.sqrt(x)):
        if (x % y == 0):
            is_prime = False
        y += 2
    if (is_prime):
        primes += 1
    x += 2
